4-projects/modules/base_env/example_peering_project.tf

```diff
@@ -61,7 +88,8 @@ module "peering_network" {
   network_name                           = "vpc-${local.env_code}-peering-base"
   shared_vpc_host                        = "false"
   delete_default_internet_gateway_routes = "true"
-  subnets                                = []
+
+  subnets = var.optional_iap_fw_rules_enabled ? [local.sb_peered] : []
```

Comment:
I think you cloud always create the sub net if you change the subnet description